axelrod/strategies/worse_and_worse.py

```diff
@@ -12,12 +13,11 @@ class WorseAndWorse(Player):
 
     Names:
         - Worse and Worse: [PRISON1998]_
-
     """
 
     name = 'Worse and Worse'
     classifier = {
-        'memory_depth': float('inf'),
```

Comment:
I'm actually not sure about this one so I've changed it back to `float('inf')` for now. The question is whether using the round number counts as using history or not, let's discuss elsewhere.